local/nginx/lua/clean_jwt_cookie.lua

```diff
@@ -5,7 +5,9 @@ if not cookie then
     ngx.log(ngx.ERR, err)
 end
 
-local ok, err = cookie:set({key = "JWT", value = ""});
+-- Set empty with an expires in the past
+local ok, err = cookie:set({key = "JWT", value = "; path=/; expires=Thu, 01 Jan 1970 00:00:00 GMT"});
```

Comment:
This was necessary for the Browser to clean the JWT cookie which was invalidated.